package.json

```diff
@@ -7,7 +7,9 @@
     "build": "tsc && cp -r views dist/",
     "devstart": "nodemon",
     "test": "jest --forceExit",
-    "lint": "tslint -p ."
+    "lint": "tslint -p .",
+    "sonarqube": "npm run coverage-jest && sonar-scanner",
+    "coverage-jest": "jest -c --forceExit --coverage"
```

Comment:
See previous comment about the target name.

Also, just curious as to what the -c flag does?

Reply:
-c is if you want to define a non default config (I should of removed this when removing jest.config.coverage.ts)